### DISCUSSION TITLE
AutoEquip Shopped Wand, AutoEquip Shopped Superior Wand, AutoEquip Shopped Triumphant Wand

### DISCUSSION THREAD

**Discussion by zylo-** [isid0re/SoloLeveling#192] [General]
Does anyone know why it Auto Shops and Auto Equips items multiple times in a row? Mine equipped 4 wands in the space of 5 seconds

**Comment by isid0re** [comment]
Its buying items better than current equipped. Then it proceeds to equip said items. Keeping the highest tiered equipped item.
